CHANGELOG.md

```diff
@@ -6,4 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [0.1.0] - 2024-02-21
-* Initial release.
\ No newline at end of file
+* Initial release.
+
+## [Unreleased]
+
+### Security
+- Add verification for invalid jumps. https://github.com/0xPolygonZero/zk_evm/pull/36
```

Comment:
This should be above, not below. And let's stick to the template (otherwise there will be too many subsections like security, documentation, testing, refactoring, misc, ...). You can merge latest main to see how it should look like.